src/index.js

```diff
@@ -11,9 +11,9 @@ type EslintContext = {
   report: Function,
   settings: ?{
     "flowtype-errors": ?{
-      flowDir: ?string
-    },
-    stopOnExit: ?any
+      flowDir: ?string,
```

Comment:
Let's prefer `flowDir?: string` instead the of `flowDir: ?string`. I believe `flowDir: ?string` allows for null types, which somewhat defeats the purpose of typechecking